src/project.js

```diff
@@ -57,7 +63,22 @@ function project(location) {
   if (typeof target === 'string') {
     newLocation.target = target;
   }
-  return newLocation;
+
+  const heightLevelList = height;
+  initIterValue += 1;
```

Comment:
What about naming this `heightLevelPosition`?

Reply:
sounds good.